### PR TITLE
Dịch "transposed-conv_vn" - Phần 1

### LINKED ISSUE #2323: Dịch "transposed-conv_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_computer-vision/transposed-conv_vn.md#L1-L114

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by tranyenthy**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2323 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

* [ ] Điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review nếu Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_computer-vision/transposed-conv_vn.md [hunk 1]
```diff
@@ -18,7 +18,12 @@ however, require to predict values for each pixel and therefore needs to increas
 Transposed convolution, also named fractionally-strided convolution :cite:`Dumoulin.Visin.2016` or deconvolution :cite:`Long.Shelhamer.Darrell.2015`, serves this purpose.
 -->
 
-*dịch đoạn phía trên*
+Các tầng mà chúng tôi đã giới thiệu gần đây về mạng nơ-ron tích chập,
+bao gồm cả các tầng tích chập (:numref:`sec_conv_layer`) và tầng gộp (:numref:`sec_pooling`), 
+thường giảm chiều rộng và chiều cao của đầu vào, hoặc giữ nguyên chúng.
+Các ứng dụng ví dụ như phân vùng theo ngữ nghĩa (:numref:`sec_semantic_segmentation`) và mạng đối sinh (:numref:`sec_dcgan`), 
+tuy nhiên, yêu cầu phải dự đoán các giá trị cho mỗi pixel vì thế cần phải tăng chiều rộng và chiều cao của đầu vào.
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:25]:
```suggestion
Tuy nhiên, các ứng dụng như phân vùng theo ngữ nghĩa (:numref:`sec_semantic_segmentation`) và mạng đối sinh (:numref:`sec_dcgan`), yêu cầu phải dự đoán các giá trị cho mỗi pixel vì thế cần phải tăng chiều rộng và chiều cao của đầu vào.
```

File: chapter_computer-vision/transposed-conv_vn.md [hunk 2]
```diff
@@ -18,7 +18,12 @@ however, require to predict values for each pixel and therefore needs to increas
...
+Tích chập chuyển vị, cũng có tên là tích chập sải bước theo phân số :cite:`Dumoulin.Visin.2016` hay phân rã tích chập :cite:`Long.Shelhamer.Darrell.2015`, phục vụ cho mục đích này.
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:26]:
```suggestion
Tích chập chuyển vị, cũng có tên là tích chập sải bước theo phân số (*fractionally-strided convolution*) :cite:`Dumoulin.Visin.2016` hay phân tách tích chập (*deconvolution*) :cite:`Long.Shelhamer.Darrell.2015`, phục vụ cho mục đích này.
```

File: chapter_computer-vision/transposed-conv_vn.md [hunk 3]
```diff
@@ -35,31 +40,33 @@ npx.set_np()
...
+## Tích chập chuyển vị 2D căn bản
 
 
 <!--
 Let us consider a basic case that both input and output channels are 1, with 0 padding and 1 stride.
 :numref:`fig_trans_conv` illustrates how transposed convolution with a $2\times 2$ kernel is computed on the $2\times 2$ input matrix.
 -->
 
-*dịch đoạn phía trên*
+Ta hãy cân nhắc một trường hợp mà các kênh đầu vào và đầu ra là 1, với đệm 0 và sải bước 1.
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:51]:
```suggestion
Ta hãy cân nhắc một trường hợp mà số kênh đầu vào và đầu ra là 1, với đệm 0 và sải bước 1. 
```

File: chapter_computer-vision/transposed-conv_vn.md [hunk 4]
```diff
@@ -35,31 +40,33 @@ npx.set_np()
...
 
 
 <!--
 Let us consider a basic case that both input and output channels are 1, with 0 padding and 1 stride.
 :numref:`fig_trans_conv` illustrates how transposed convolution with a $2\times 2$ kernel is computed on the $2\times 2$ input matrix.
 -->
 
-*dịch đoạn phía trên*
+Ta hãy cân nhắc một trường hợp mà các kênh đầu vào và đầu ra là 1, với đệm 0 và sải bước 1. 
+:numref:`fig_trans_conv` mô tả cách tích chập chuyển vị có một hạt nhân $2\times 2$ được tính toán trên một ma trận đầu vào kích thước $2\times 2$.
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:52]:
```suggestion
:numref:`fig_trans_conv` mô tả cách tích chập chuyển vị với một hạt nhân $2\times 2$ được tính toán trên một ma trận đầu vào kích thước $2\times 2$.
```

File: chapter_computer-vision/transposed-conv_vn.md [hunk 5]
```diff
@@ -35,31 +40,33 @@ npx.set_np()
...
+
 
 
 <!--
 ![Transposed convolution layer with a $2\times 2$ kernel.](../img/trans_conv.svg)
 -->
 
 
-![*dịch mô tả phía trên*](../img/trans_conv.svg)
+![Tầng tích chập chuyển vị có một hạt nhân $2\times 2$.](../img/trans_conv.svg)
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:61]:
```suggestion
![Tầng tích chập chuyển vị với một hạt nhân $2\times 2$.](../img/trans_conv.svg)
```

File: chapter_computer-vision/transposed-conv_vn.md [hunk 6]
```diff
@@ -79,14 +86,15 @@ Remember the convolution computes results by `Y[i, j] = (X[i: i + h, j: j + w] *
 While the transposed convolution broadcasts input values through the kernel, which results in a larger output shape.
 -->
 
-*dịch đoạn phía trên*
+Hãy nhớ ràng tích chập tính toán kết quả bằng `Y[i, j] = (X[i: i + h, j: j + w] * K).sum()` (tham khảo theo `corr2d` trong :numref:`sec_conv_layer`), tổng hợp các giá trị đầu vào thông qua hạt nhân.
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:89]:
```suggestion
Hãy nhớ rằng kết quả tính tích chập là `Y[i, j] = (X[i: i + h, j: j + w] * K).sum()` (tham khảo theo `corr2d` trong :numref:`sec_conv_layer`), tức tổng hợp các giá trị đầu vào thông qua hạt nhân.
```

File: chapter_computer-vision/transposed-conv_vn.md [hunk 7]
```diff
@@ -101,7 +109,8 @@ Or we can use `nn.Conv2DTranspose` to obtain the same results.
 As `nn.Conv2D`, both input and kernel should be 4-D tensors.
 -->
 
-*dịch đoạn phía trên*
+Hoặc  ta có thể sử dụng `nn.Conv2DTranspose` để thu được kết quả tương tự.
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:112]:
```suggestion
Hoặc ta có thể sử dụng `nn.Conv2DTranspose` để thu được kết quả tương tự.
```

File: chapter_computer-vision/transposed-conv_vn.md [hunk 8]
```diff
@@ -101,7 +109,8 @@ Or we can use `nn.Conv2DTranspose` to obtain the same results.
...
+Là `nn.Conv2D`, cả đầu vào và hạt nhân phải là tensor 4 chiều.
```
Comment (cuongvng) [inline chapter_computer-vision/transposed-conv_vn.md:113]:
```suggestion
Vì là `nn.Conv2D`, cả đầu vào và hạt nhân phải là tensor 4 chiều.
```
